test/topology/test_replace.py

```diff
@@ -48,3 +49,17 @@ async def test_replace_reuse_ip_using_host_id(manager: ManagerClient) -> None:
     replace_cfg = ReplaceConfig(replaced_id = servers[0].server_id, reuse_ip_addr = True, use_host_id = True)
     await manager.server_add(replace_cfg)
     await wait_for_token_ring_and_group0_consistency(manager, time.time() + 30)
+
+@pytest.mark.asyncio
+async def test_replacing_alive_node_fails(manager: ManagerClient) -> None:
```

Comment:
I'd suggest to enable this test only in a single mode like `dev` to save some resources, we don't really exercise some critical paths here so no need to run it in `debug`.

Reply:
Right, this test is also not the fastest. I'll move it to a separate file and make it `dev` only.